### PR TITLE
add VECTOR doc

### PR DESCRIPTION
**Pull request by taroface**
DOC-10634

### CODE REVIEW

**Comment by jordanlewis** [comment]
This looks correct from an Engineering perspective. Perhaps @mgartner would want to weigh in as well.

From the product perspective, I think we should consider adding some more exposition about how to use the feature for a simple embedding search with common data, such as embeddings from OpenAI. But I will let @dikshant weigh in on that.

**Comment by dikshant** [comment]
> I think we should consider adding some more exposition about how to use the feature for a simple embedding search with common data, such as embeddings from OpenAI

We will cover some of this in an upcoming blog post (targeted for mid September) and we can update the docs then.

**Review by dikshant — APPROVED** [review]
LGTM

**Review by jordanlewis — APPROVED** [review]

**Review by florence-crl — APPROVED** [review]
lgtm pending adding enterprise note

File: src/current/v24.2/vector.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+Create a table with a `VECTOR` column, specifying `3` dimensions:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE TABLE items (
+    category STRING,
+    vector VECTOR(3),
+    INDEX (category)
+);
```
[the remark this answers is not in the document]
Comment (taroface) [reply src/current/v24.2/vector.md:62]: Good catch!!!!

**Pull request merged**